rotkehlchen/tests/unit/test_assets.py

```diff
@@ -21,23 +20,18 @@
 def test_unknown_asset():
     """Test than an unknown asset will throw"""
     with pytest.raises(UnknownAsset):
-        Asset('jsakdjsladjsakdj')
+        FiatAsset('jsakdjsladjsakdj')
 
 
 def test_asset_nft():
     a = Asset('_nft_foo')
     assert a.identifier == '_nft_foo'
-    assert a.symbol is not None
-    assert a.name == 'nft with id _nft_foo'
-    assert a.started is not None
-    assert a.forked is None
-    assert a.swapped_for is None
-    assert a.cryptocompare is not None
-    assert a.coingecko is None
+    should_not_exist = {'name', 'symbol', 'started', 'forked', 'swapped_for', 'cryptocompare', 'coingecko'}  # noqa: E501
+    assert 0 == len(should_not_exist & a.__dict__.keys())
```

Comment:
nitpick. I really dislike the yoda notation

```suggestion
    assert len(should_not_exist & a.__dict__.keys()) == 0
```